packages/office-ui-fabric-react/src/components/Pivot/Pivot.styles.ts

```diff
@@ -68,14 +59,18 @@ const linkStyles = (props: IPivotStyleProps): IStyle[] => {
         },
         [`.${IsFocusVisibleClassName} &:focus`]: {
           outline: `1px solid ${semanticColors.focusBorder}`
+        },
+        [`.${IsFocusVisibleClassName} &:focus:after`]: {
+          content: 'attr(data-content)',
+          position: 'relative',
+          border: 0
         }
       }
     },
     rootIsLarge && {
       fontSize: FontSizes.large
     },
     rootIsTabs && [
-      getFocusStyle(theme),
```

Comment:
Is there any way to make this work while keeping this call in place? It's great to have the Pivot not jump around, but I worry about removing this call that gives users hooks to style a focused tab. I believe it will break some existing scenarios.

Reply:
The issue with the getFocusStyle function is that its styling is applied with a higher priority than any other styling in this file.
If you look into the getFocusStyle() function you'll see that it defines the following properties:
`content: '""',
 position: 'absolute',
 border: '1px solid ' + borderColor,`

- the content and position attribute both screw up the :after elements logic and position and 
- the border changes the element's width

This is not only true for the rootIsTabs element, but for the linkIsSelected style as well (here it is more hidden, as it is applied through the CommandBarButton and the BaseButton element).

To prevent the properties from the getFocusStyle function to change the elements width, for the linkIsSelected element these styles are overwritten explicitly in l.63-66.

We could keep the getFocusStyle call here in the rootIsTabs, but then we need to add this explicit overwrite of the 3 properties to the rootIsTabs element as well (which leads to adding and overwriting the styles right after another). 
This would keep the zIndex and HighContrastSelector styles from the getFocusStyle intact.


Apart from that:
I saw that there is another [PR](https://github.com/OfficeDev/office-ui-fabric-react/pull/8640) ongoing to fix the getFocusStyle function, which looks like it allows adjusting the styles (which we could use here). 
But until this is merged there are only 2 options to fix the styling problem: either overwrite or don't call the getFocusStyle function.


What do you think is the best option?

Reply:
@jdhuntington It would be great if you could review Ute her comment above :)